ops/testing.py

```diff
@@ -325,10 +328,10 @@ def _create_meta(self, charm_metadata, action_metadata):
 
         return charm.CharmMeta.from_yaml(charm_metadata, action_metadata)
 
-    def _load_config_defaults(self, charm_config):
-        """Load default values from config.yaml.
+    def _get_config(self, charm_config: typing.Optional[dict]):
+        """If the user passed a config to Harness, use it.
 
-        Handle the case where a user doesn't supply explicit config snippets.
+        Otherwise, attempt to load one from charm_dir/config.yaml.
         """
         filename = inspect.getfile(self._charm_cls)
         charm_dir = pathlib.Path(filename).parents[1]
```

Comment:
I'm struggling to see how adding this extra method improves things here.  It's only used in one place and its result is just passed directly into "_load_config_defaults".